packages/teams-js/src/public/video.ts

```diff
@@ -216,4 +216,92 @@ export namespace video {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.video ? true : false;
   }
+
+  /**
+   * Namespace to get video frames from a media stream
+   */
+  export namespace mediaStream {
+    export function isSupported(): boolean {
+      return ensureInitialized(runtime) && textureStreamAvailable() && !!runtime.supports.video?.mediaStream;
+    }
+
+    function textureStreamAvailable(): boolean {
+      return !!(window['chrome']?.webview?.getTextureStream && window['chrome']?.webview?.registerTextureStream);
```

Comment:
all references to window should be surrounded by a check like this:

`if (typeof window === 'undefined')`

That will make sure that if your function is called in a server-side rendered environment that it won't cause a break.

Reply:
added checking